src/routes/process-user-vocabulary/UserVocabularyProcessScreen.tsx

```diff
@@ -118,13 +118,7 @@ const UserVocabularyProcessScreen = ({ navigation, route }: UserVocabularyProces
         await incrementNextUserVocabularyId()
       }
 
-      const imagePaths = await Promise.all(
-        images.map(async (image, index) => {
-          const path = `file:///${DocumentDirectoryPath}/image-${id}-${index}.jpg`
-          await moveFile(image, path)
-          return { id: index, image: path }
-        })
-      )
+      const imagePaths = images.map((image, index) => ({ id: index, image }))
```

Comment:
You cannot remove this part, as it is really important.
We move these images as when they are taken in the app, they are only saved to a cache, to persist them and see them later, we need to save them to a permanent path. You can see this error, when you create an own vocabulary, then close the app, clear the app cache and reopen the app, then the image of this word is not available. 
As this seems to be related to the sentry bug, you should investigate, why the id counter does not work properly and this bug occurs.